### ISSUE TITLE
Linking to anchors in the documentation

### ISSUE THREAD

**Issue by madx** [esdoc/esdoc#80]
I can't link to an anchor on the same documentation page. Here's and example of what I'm trying to achieve

``` javascript
/**
 * This is the documentation for my class.
 * 
 * [A link to another part of this documentation](#title)
 * 
 * [...]
 *
 * ## Title
 */
export default class Foo {}
```

The markdown parser correctly generates `<h2 id="title">Title</h2>` but the link to this anchor is made against the root of the documentation (`<a href="/#title">`).

Maybe there's an option for selecting the kind of links to generate? (Absolute or relative)

**Comment by madx** [comment]
I found that this is due to the `<base>` tag in the template. Do you think there is a way to remove it somehow?

**Comment by h13i32maru** [comment]
@madx Hi! Thanks for this issue. Sorry for late reply.
I fixed this problem in b1c9e67. You can use inner link:

```markdown
# Inner Link
...
[foo](#inner-link)
```
